python/hail/dataset.py

```diff
@@ -3009,6 +3011,10 @@ def lmmreg(self, kinshipMatrix, y, covariates=[], global_root="global.lmmreg", v
 
         FastLMM uses the Realized Relationship Matrix (RRM) for kinship. This can be computed with :py:meth:`~hail.VariantDataset.rrm`. However, any instance of :py:class:`KinshipMatrix` may be used, so long as ``sample_list`` contains the complete samples of the caller variant dataset in the same order.
 
+        **LD Matrix**
+
+        When using less variants than samples, it becomes wasteful to compute a gigantic RRM matrix considering that it will end up being low rank and as such contain at most m nonzero eigenvalues. For this reason, this method can take an LD Matrix instead of a Kinship Matrix. The eigendecomposition of this LD Matrix will then be used to compute the eigenvalues and eigenvectors that would have been returned from the RRM.
```

Comment:
We should post a script demonstrating LOCO, looping over chromosomes, where we just use a chromosome filter and its complement as this parameter and as a filter on the LD matrix respectively. For large n this is cleaner than computing 22 kinship matrices, even if that's done slickly.